sanic/router.py

```diff
@@ -15,6 +14,8 @@
     'alpha': (str, r'[A-Za-z]+'),
 }
 
+ROUTER_CACHE_SIZE = 1024
```

Comment:
Because it actually cannot be modified. The value is read and applied only once when the sanic module is imported, any further changes don't have any effect.